default/scripting/policies/RACIAL_PURITY.focs.txt

```diff
@@ -9,25 +9,68 @@ Policy
     effectsgroups = [
         [[SPECIES_LIKES_OR_DISLIKES_POLICY_STABILITY_EFFECTS]]
         
+        // This policy disables influence and research foci for non-capital, non-exobot species
+        // See /species/common/focus.macros
+
+        // Creates a special to ensure that deadopting this policy has bad consequencies for a while.
+        EffectsGroup
+            scope = Source
```

Comment:
Did not code many policies - so what is in the scope? the EmpireSource?

Reply:
I hope Source is the policy or the empire. In any case, this creates the special at Capital.

Reply:
Ok, in that case  it is the EmpireSource (i.e. Capital or a random planet if there is no capital)